### DISCUSSION TITLE
Quick Solder Board for SNES - Progress

### DISCUSSION THREAD

**Discussion by CustomZ02** [darthcloud/BlueRetro#757] [Show and tell]
So I’m on version2 of my design, been doing some testing. Had one faulty esp32, replaced and can now register my ps4/5 controllers. But I’m having an issue where it’s skipping Player 1 and syncing player 2,3, and 4 just fine. I can’t seem to get it to register player 1. Is there something I’m missing? 

I tried looking thru the web interface to see if maybe a setting was causing it to skip player 1 but I can’t seem to find anything. Currently controllers light up in order Red,Green, and Pink completely bypassing Blue for player one. Anyone have any pointers as to what I may have missed?

Thanks in Advace!

![IMG_0122](https://github.com/darthcloud/BlueRetro/assets/143313315/f1d9e6d9-eb00-403a-a39c-63d067bfe49c)

![IMG_0135](https://github.com/darthcloud/BlueRetro/assets/143313315/9c263530-2340-4bcd-b092-9ca4d8788acd)

![IMG_0131](https://github.com/darthcloud/BlueRetro/assets/143313315/c8ba7e50-6a32-4eb1-aab4-0ec52d0c49a1)

![IMG_0121](https://github.com/darthcloud/BlueRetro/assets/143313315/14e9b090-42ba-4d52-944d-df3848d45fa6)

**Comment by darthcloud** [comment]
Maybe you got a rogue BT device connecting as player 1?

Like BLE light or BLE speaker

Try to get the serial output.

**Reply by darthcloud** [reply]
Also try only HW1 firmware first, just to know if the basic stuff work first.

**Reply by CustomZ02** [reply]
Going to load up HW1 and see if that helps, thanks for the quick response.

**Reply by CustomZ02** [reply]
Seems like something was up with the esp32. Replaced it and controllers connect properly now. Sorry for the late reply. 

![IMG_1153](https://github.com/darthcloud/BlueRetro/assets/143313315/f8173cb3-63f1-4129-ba36-b94d37ae9875)

**Comment by kaduleite** [comment]
Does your board have input detection? Are you going to sell or share the project?
